reinforcement_learning/bindings/cs/rl.net.cli/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.2" />
+    </startup>
+</configuration>
```

Comment:
[nit] general comment applicable to almost all files: missing new line at the end (to make POSIX happy)